### ISSUE TITLE
Notification that someone is working on that "Haushalt"

### ISSUE THREAD

**Issue by monsieurhannes** [okfde/offenerhaushalt.de#101]
An interesting feature would be to tell when you're working on a Haushalt. The most simple would be to at least inform about the data that got uploaded, as in a little "pending" notice next to the city in the list. Another, bigger project would be to say that you're currently working on one Haushalt and add a email-address for contacting. The "claim" could have a certain lifetime - e.g. 30 days. If it doesn't get published within that time the notification vanishes.
All just ideas, I have no clue how to actually program it.

**Comment by michaeljokf** [comment]
yes that might be a way to go!  We will work more on the responsiveness of the adding data dialogue.
